package.json

```diff
@@ -24,6 +24,7 @@
     "Marco Perego (https://github.com/emmelleppi)",
     "Josh Ellis (https://github.com/joshuaellis)"
   ],
+  "type": "module",
```

Comment:
added `"type": "module"` because rollup was complaining about reading rollup.config.js as cjs while it is esm.

Alternatively, I could have renamed it to `rollup.config.mjs` but I think `"type": "module"` is ok for the published module, since we have `main` (for cjs) and `module` defined.

Though I haven't test with the resulting built module